Source/WebCore/cssjit/SelectorCompiler.cpp

```diff
@@ -1337,6 +1338,10 @@ static inline FunctionType addPseudoClassType(const CSSSelector& selector, Selec
         fragment.languageArgumentsList.append(selector.argumentList());
         return FunctionType::SimpleSelectorChecker;
 
+    case CSSSelector::PseudoClassType::State:
+        fragment.customStateSet.append(selector.argument());
+        return FunctionType::SimpleSelectorChecker;
+
```

Comment:
Right now, we're populating this data and not using it. This is supposed to be used for CSSJIT, see: https://github.com/WebKit/WebKit/commit/57ad5510bfa05b8679d85c9becf7819e62854a1e

You should either implement CSSJIT support fully or remove the `fragment.customStateSet` field and set the pseudo class to `FunctionType::CannotCompile`

Reply:
Ah I thought this was sufficient, but now I see it's a little more complex I'll skip it for this PR and perhaps come back to add it in another PR.